gdx-backend-jtransc/src/com/badlogic/gdx/graphics/Pixmap.java

```diff
@@ -419,11 +431,25 @@ public void drawPixmap(Pixmap pixmap, int x, int y) {
 	// copyPixels (sourceImage:Image, sourceRect:Rectangle, destPoint:Vector2, alphaImage:Image = null, alphaPoint:Vector2 = null, mergeAlpha:Bool = false):Void
 	public void drawPixmap(Pixmap pixmap, int x, int y, int srcx, int srcy, int srcWidth, int srcHeight) {
 		ensureLoaded();
-		for (int my = 0; my < srcHeight; my++) {
-			for (int mx = 0; mx < srcWidth; mx++) {
-				this._setPixel(x + mx, y + my, pixmap._getPixel(srcx + mx, srcy + my));
-			}
+		int byteWidth = width << 2;
```

Comment:
I think you can multiply by 4 safely. All compilers optimize integer multiplications and divisions by powers of two converting them into shifts. And if it is not the case I can always optimize it at jtransc level.
At any rate, just a comment, not really important. I can understand and byteWifth is descriptive enough